sonic-syseepromd/scripts/syseepromd

```diff
@@ -115,6 +122,59 @@ class DaemonSyseeprom(daemon_base.DaemonBase):
 
         return True
 
+    def post_values_to_state_db(self, key, values):
+        try: 
+            state_db = daemon_base.db_connect(STATE_DB)
+            for k,v in values.items():
+                state_db.hset(key, k, str(v))
```

Comment:
is it possible to use hmset here? hmset may get better performance and synchronization.

Reply:
hmset has different API now after replacing swsssdk with swsscommon, can you give me an example how are you using it?